topics/releases.md

```diff
@@ -33,6 +33,13 @@ The following table lists details of the latest Ktor releases.
 
 <table>
 <tr><td>Version</td><td>Release Date</td><td>Highlights</td></tr>
+<tr><td>3.1.2</td><td>March 27, 2025</td><td><p>
+A patch release that updates Kotlin to 2.1.20 and removes empty artifacts. It also fixes various issues, including Base64 
```

Comment:
> and removes empty artifacts

I think it isn't worth mentioning, as it gives no value to our users, it's rather a technical change.